locale/fi/LC_MESSAGES/django.po

```diff
@@ -2623,23 +2624,97 @@ msgstr ""
+msgstr "Valitse moduulit: "
+
+#: exercise/templates/exercise/staff/results.html:70
+msgid "Select exercises: "
+msgstr "Valitse tehtävät: "
+
+#: exercise/templates/exercise/staff/results.html:84
 msgid ""
-"\n"
-"                \tStudent (N=%(num)s)\n"
-"                \t"
+"Total number of submissions. Calculates all student submission counts "
+"together."
 msgstr ""
-"\n"
-"Opiskelija (N=%(num)s)"
+"Palautusten lukumäärä. Laskee kaikkien opiskelijoiden kaikkien palautusten "
+"lukumäärän yhteen."
+
+#: exercise/templates/exercise/staff/results.html:85
+msgid "Total submissions"
+msgstr "Palautuksia yhteensä"
+
+#: exercise/templates/exercise/staff/results.html:89
+msgid ""
+"How many submissions a single student has used on the exercise on average. "
+"Only accounts for students with one or more submissions."
+msgstr ""
+"Kuinka monta palautusta opiskelija on käyttänyt tehtävään keskimäärin. "
+"Lasketaan vain opiskelijat, joilla on vähintään yksi palautus."
+
+#: exercise/templates/exercise/staff/results.html:91
+msgid "Average submissions per student"
+msgstr "Palautuskertojen keskiarvo per opiskelija"
+
+#: exercise/templates/exercise/staff/results.html:95
+msgid "Maximum number of available submissions for the exercise."
+msgstr "Kuinka monta palautusta tehtävässä on mahdollista käyttää."
+
+#: exercise/templates/exercise/staff/results.html:96
+msgid "Max submissions"
+msgstr "Palautusmahdollisuuksia"
+
+#: exercise/templates/exercise/staff/results.html:100
+msgid "Number of students that have one or more exercise submissions."
+msgstr "Opiskelijat, joilla on vähintään yksi palautus."
+
+#: exercise/templates/exercise/staff/results.html:102
+msgid "Students with submissions"
+msgstr "Tehtävän palauttaneet opiskelijat"
+
+#: exercise/templates/exercise/staff/results.html:106
+msgid "Number of students that have received maximum points from the exercise."
+msgstr "Opiskelijat, jotka ovat saaneet täydet pisteet tehtävästä."
 
-#: exercise/templates/exercise/staff/results.html:28
-msgid "Sum"
-msgstr "Yhteensä"
+#: exercise/templates/exercise/staff/results.html:107
+msgid "Students with max points"
+msgstr "Opiskelijat, joilla on täydet pisteet"
 
-#: exercise/templates/exercise/staff/results.html:44
-msgid "Maximum points"
-msgstr "Enimmäispisteet"
+#: exercise/templates/exercise/staff/results.html:111
+msgid ""
+"Average points received for the exercise per student. Only accounts for "
+"students with one or more submissions."
+msgstr ""
+"Kuinka monta pistettä opiskelijat ovat saaneet tehtävästä keskimäärin. "
+"Lasketaan vain opiskelijat, joilla on vähintään yksi palautus."
+
+#: exercise/templates/exercise/staff/results.html:112
+msgid "Average points per student"
+msgstr "Pisteiden keskiarvo per opiskelija"
+
+#: exercise/templates/exercise/staff/results.html:116
+msgid "Maximum points for the exercise."
+msgstr "Tehtävän maksimipisteet."
+
+#: exercise/templates/exercise/staff/results.html:117
+msgid "Max points"
```

Comment:
Should this be `Maximum points` as it was before?

Reply:
Changed to the suggestions